src/features/query-bar/components/filter/Condition.js

```diff
@@ -176,12 +178,52 @@ const Condition = ({
         );
     };
 
+    const renderAttributeField = () => {
```

Comment:
Generally, anything in a `renderThing` type function should probably be its own component. Such a change is outside of the scope of this PR, so don't worry about changing it now.